src/color.rs

```diff
@@ -131,16 +128,24 @@ impl Elem {
 pub type ColoredString = StyledContent<String>;
 
 pub struct Colors {
-    theme: Option<Theme>,
+    theme: Option<ColorTheme>,
     lscolors: Option<LsColors>,
 }
 
 impl Colors {
     pub fn new(t: ThemeOption) -> Self {
         let theme = match t {
             ThemeOption::NoColor => None,
-            ThemeOption::Default | ThemeOption::NoLscolors => Some(Theme::default()),
-            ThemeOption::Custom(ref file) => Some(Theme::from_path(file).unwrap_or_default()),
+            ThemeOption::Default | ThemeOption::NoLscolors => Some(Theme::default().color),
+            ThemeOption::Custom(ref file) => {
```

Comment:
I guess the only additional thing that we would need to support https://github.com/Peltoche/lsd/issues/749 would be to add an additional check here in case filename is not mentioned but we can take that up in another PR.

Reply:
https://github.com/Peltoche/lsd/issues/749 updated